cartridges/int_bolt_embedded_sfra/cartridge/controllers/Order.js

```diff
@@ -85,14 +85,16 @@ server.replace(
                 returningCustomer: false,
                 passwordForm: passwordForm,
                 reportingURLs: reportingURLs,
-                orderUUID: order.getUUID()
+                orderUUID: order.getUUID(),
+                boltNewCreatedCustomer: !!skipCustomerCheck
```

Comment:
Do we need to hide that 'save information' section also for Bolt logged-in shoppers? I think this only takes care of guest shoppers that created an account during checkout?

Reply:
nvm, just noticed logged in shopper doesn't have that register section anyways